### PR TITLE
Amend displaying breeds in select-breed page

### PR DESCRIPTION
**Pull request by zdzislawbela**

### CODE REVIEW

File: components/BreedCheckbox/BreedCheckbox.tsx [hunk 1]
```diff
@@ -16,6 +16,15 @@ export const BreedCheckbox = ({ breed, handleCheckbox, isChecked }: Props) => {
     handleCheckbox(event.target.value);
   };
 
+  const formatBreed = () => {
+    if (!breed.includes("/")) return breed;
+
+    return `${breed.slice(0, breed.indexOf("/"))}-${breed.slice(
+      breed.indexOf("/") + 1,
+      breed.length
+    )}`;
```
Comment (raqso) [inline components/BreedCheckbox/BreedCheckbox.tsx]: Not much readable. What about something like this:

```suggestion
	const [subBreed, breed] = breed.split('/');
    return `${subBreed} ${breed}`;
```

File: components/BreedCheckbox/BreedCheckbox.tsx [hunk 2]
```diff
@@ -16,6 +16,15 @@ export const BreedCheckbox = ({ breed, handleCheckbox, isChecked }: Props) => {
     handleCheckbox(event.target.value);
   };
 
+  const formatBreed = () => {
```
Comment (raqso) [inline components/BreedCheckbox/BreedCheckbox.tsx]: This function doesn't have to be in the component scope and can take breed from its own parameter.

**Review by raqso — CHANGES_REQUESTED** [review]

File: components/helpers/formatBreed.ts [hunk 3]
```diff
@@ -0,0 +1,11 @@
+export const formatBreed = (breed: string) => {
+  const [mainBreed, subBreed] = breed.split("/");
+
+  if (!subBreed) {
+    return breed.charAt(0).toUpperCase() + breed.slice(1);
```
Comment (raqso) [inline components/helpers/formatBreed.ts:5]: 2 things:

1. You've got this part of code repeated, remember about the DRY principle,
2. I think such helper function like 'capitalize(text: string)' would be nice to have in helpers

**Review by raqso — CHANGES_REQUESTED** [review]

File: components/helpers/formatBreed.ts (hunk 3 above)
Comment (zdzislawbela) [reply components/helpers/formatBreed.ts:5]: good point. next commit is covering it, but suddenly after this branch is merged :hankey:

**Pull request merged**